docs/usage/request_default_domain_cert.md

```diff
@@ -0,0 +1,133 @@
+
+> Your application should be able to reload replaced certificates in a timely manner to avoid service disruptions.
+
+Certificates can be requested via 3 resources type
+- Ingress
+- Service (type LoadBalancer)
+- certificate (Gardener CRD)
+
+If either of the first 2 are used, a corresponding `Certificate` resource will automatically be created.
+
+### Using an ingress Resource
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: amazing-ingress
+  annotations:
+    cert.gardener.cloud/purpose: managed
+    #cert.gardener.cloud/issuer: custom-issuer
+spec:
+  tls:
+  - hosts:
+    # Must not exceed 64 characters.
+    - short.ingress.shoot.project.default-domain.gardener.cloud
+    # Certificate and private key reside in this secret.
+    secretName: testsecret-tls
+  rules:
+  - host: short.ingress.shoot.project.default-domain.gardener.cloud
+    http:
+      paths:
+      - pathType: Prefix
+        path: "/"
+        backend:
+          service:
+            name: amazing-svc
+            port:
+              number: 8080
+```
+
+### Using a service type LoadBalancer
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    cert.gardener.cloud/purpose: managed
+    # Certificate and private key reside in this secret.
+    cert.gardener.cloud/secretname: test-service-secret
+    # You may add more domains separated by commas (e.g. "service.shoot.project.default-domain.gardener.cloud, amazing.shoot.project.default-domain.gardener.cloud")
+    dns.gardener.cloud/dnsnames: "service.shoot.project.default-domain.gardener.cloud" 
+    dns.gardener.cloud/ttl: "600"
+    #cert.gardener.cloud/issuer: custom-issuer
+  name: test-service
+  namespace: default
+spec:
+  ports:
+    - name: http
+      port: 80
+      protocol: TCP
+      targetPort: 8080
+  type: LoadBalancer
+```
+
+### Using the custom Certificate resource
+```yaml
+apiVersion: cert.gardener.cloud/v1alpha1
+kind: Certificate
+metadata:
+  name: cert-example
+  namespace: default
+spec:
+  commonName: short.ingress.shoot.project.default-domain.gardener.cloud
+  secretRef:
+    name: cert-wildcard
+    namespace: default
+  # Optionnal if using the default issuer
+  issuerRef:
+    name: garden
+```
+
+If you're interested in the current progress of your request, you're advised to consult the description, more specifically the `status` attribute in case the issuance failed.
+
+## Request a wildcard certificate
+In order to avoid the creation of multiples certificates for every single endpoints, you may want to create a wildcard certificate for your shoot's default cluster.
+
+```yaml
+apiVersion: cert.gardener.cloud/v1alpha1
+kind: Certificate
```

Comment:
same as above.
Many users just use the same wildcard domain name in their ingress and service annotations, i.e. using the `cert.gardener.cloud/commonname` annotation. It has the same effect and may be more convenient and declarative. If multiple ingress or service annotations use exactly the same domain name(s) for the certificate, the certificate is only requested once.
Maybe at least mention it as an alternative?

Reply:
Good point, changed it to an Ingress